lxd/resources/utils.go

```diff
@@ -124,3 +127,83 @@ func udevDecode(s string) (string, error) {
 
 	return ret, nil
 }
+
+func pciAddress(devicePath string) (string, error) {
+	// Check if we have a subsystem listed at all.
+	if !sysfsExists(filepath.Join(devicePath, "subsystem")) {
+		return "", nil
+	}
+
+	// Track down the device.
+	linkTarget, err := filepath.EvalSymlinks(devicePath)
+	if err != nil {
+		return "", errors.Wrapf(err, "Failed to track down \"%s\"", devicePath)
```

Comment:
@stgraber just a general observation, but any reason you dont use the `%q` for double quoting strings rather than `\"%s\"`?

Reply:
Not really, we probably should do that across all of `lxd/resources`, we have a bunch of those ;)

Reply:
Yeah. I also tend to use backticks for go strings when I need to put double quotes in the string rather than escaping, as I find it easier to read.

Reply:
Done